src/newrelic/infinite_tracing/trace_observer.go

```diff
@@ -358,36 +363,58 @@ func (to *TraceObserver) handleSupportability() {
 			// alive when a shutdown was triggered from the sender.
 			return
 		case inc := <-to.supportability.increment:
-			metrics[inc.name] += inc.count
+			v, ok := metrics[inc.name]
+			if !ok {
+				v = [6]float64{0.0, 0.0, 0.0, 0.0, 0.0, 0.0}
```

Comment:
Technically, v[0]/Count is a long, but looks like this is how we've been doing it all along?

It'd be useful to either have helper functions to update the the supportability usage, or since elements are being modified directly, a comment clarifying all the parts of v ( like interaction count, bytes sent, bytes received, unused, unused, unused).

Reply:
Yes, all of our metrics are stored as float64's.

Reply:
Addressed the helper and comment here [chore(daemon): add clarifying comment and refactor code duplication](https://github.com/newrelic/newrelic-php-agent/pull/507/commits/03576673bebf12c2a2835c985936a9a5fe5ecd76)